source/common/router/router.h

```diff
@@ -48,6 +48,27 @@ namespace Router {
   COUNTER(rq_retry_skipped_request_not_complete)
 // clang-format on
 
+/*
+ * A FilterState object that tracks a single uint32_t value.
+ */
+class UInt32Accessor : public StreamInfo::FilterState::Object {
```

Comment:
I think this is useful enough to merit being located near stream info (and unit test!)

Reply:
Done. Moved this under */stream_info. I think it may make sense to move Route::StringAccessor* as well, but that's probably a separate PR. Let me know if you want me to do it here.